snapcraft/internal/repo.py

```diff
@@ -576,3 +568,18 @@ def _set_pkg_version(pkg, version):
         pkg.candidate = version
     else:
         raise PackageNotFoundError('{}={}'.format(pkg.name, version))
+
+
+_libc_lib_list = None
+
+
+def libc_library_list():
```

Comment:
Yeah, if you see this being more generally useful, sure. Still want me to save the result, just make it a dict instead?